src/run/run.c

```diff
@@ -1132,19 +1132,12 @@ static int transient_service_set_properties(sd_bus_message *m, const char *pty_p
                         return bus_log_create_error(r);
         }
 
-        if (pty_path) {
-                _cleanup_close_ int pty_slave = -EBADF;
-
-                pty_slave = open_terminal(pty_path, O_RDWR|O_NOCTTY|O_CLOEXEC);
-                if (pty_slave < 0)
-                        return pty_slave;
-
+        if (pty_slave != -EBADF) {
                 r = sd_bus_message_append(m,
-                                          "(sv)(sv)(sv)(sv)",
+                                          "(sv)(sv)(sv)",
                                           "StandardInputFileDescriptor", "h", pty_slave,
                                           "StandardOutputFileDescriptor", "h", pty_slave,
-                                          "StandardErrorFileDescriptor", "h", pty_slave,
-                                          "TTYPath", "s", pty_path);
```

Comment:
I'd like to keep `TTYPath=` here. While it's not used by sd-executor, it would be nice to show this in unit property.

Reply:
yes, this really should stay, we use this for other purposes too, for example informational stuff, and utmp entries and so on.